service/statistics/AnalyticsEvents.js

```diff
@@ -216,6 +216,14 @@ export const ICE_STATE_CHANGED = 'ice.state.changed';
  */
 export const TRACK_UNMUTED = 'track.unmuted';
 
+/**
+ * Indicates that a track has video blur enabled
+ *
+ * Properties:
+ *      value: 'enabled' or 'disabled'
+ */
+export const TRACK_VIDEOBLUR = 'track.videoblur';
```

Comment:
Why don't you create an event creator function?

Reply:
I used Statistics.sendAnalytics as I saw it being used without a creator function - in RTCUtils.js